secrets-provider/values.schema.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "required": [
+    "rbac",
+    "secretsProvider",
+    "environment"
+  ],
+  "properties": {
+    "rbac": {
+      "required": [
+        "create",
+        "roleName",
+        "roleBindingName",
+        "serviceAccount"
+      ],
+      "properties": {
+        "create": {
+          "type": "boolean"
+        },
+        "roleName": {
+          "type": "string",
+          "minLength": 1
+        },
+        "roleBindingName": {
+          "type": "string",
+          "minLength": 1
+        },
+        "serviceAccount": {
+          "required": [
+            "name"
+          ],
+          "properties": {
+            "name": {
+              "type": "string",
+              "minLength": 1
+            }
+          }
+        }
+      }
+    },
+    "secretsProvider": {
+      "required": [
+        "image",
+        "tag",
+        "pullPolicy",
+        "name"
+      ],
+      "properties": {
+        "image": {
+          "type": "string",
+          "minLength": 1
+        },
+        "tag": {
+          "type": ["string", "number"],
+          "minLength": 1,
+          "pattern": "^\\d+(\\.\\d+){0,2}$"
+        },
+        "pullPolicy": {
```

Comment:
Since in K8s, the parameter name is `imagePullPolicy`, I think it would be nice to keep the same name to provide a known language the the user already familiar with.

Reply:
you got it!